app/actions/views/channel.js

```diff
@@ -136,22 +138,22 @@ export function loadProfilesAndTeamMembersForDMSidebar(teamId) {
     };
 }
 
-export function loadPostsIfNecessary(channel) {
+export function loadPostsIfNecessary(channelId) {
     return async (dispatch, getState) => {
         const state = getState();
         const {posts, postsInChannel} = state.entities.posts;
 
-        const postsIds = postsInChannel[channel.id];
+        const postsIds = postsInChannel[channelId];
 
         // Get the first page of posts if it appears we haven't gotten it yet, like the webapp
-        if (!postsIds || postsIds.length < Posts.POST_CHUNK_SIZE) {
-            return getPosts(channel.id)(dispatch, getState);
+        if (!postsIds || postsIds.length < POST_INCREASE_AMOUNT) {
```

Comment:
I'm following the same path as the webapp and one other thing is that this way we can control the amount of posts to render at a time